tests/acceptance/TestHelpers/GraphHelper.php

```diff
@@ -1051,7 +1051,8 @@ public static function getMySpaces(
 		array  $body = [],
 		array  $headers = []
 	): ResponseInterface {
-		$url = self::getFullUrl($baseUrl, 'me/drives' . $urlArguments);
+		$urlArguments = $urlArguments ? "?$urlArguments" : "";
+		$url = self::getFullUrl($baseUrl, "me/drives$urlArguments");
```

Comment:
```suggestion
		$url = self::getFullUrl($baseUrl, "me/drives" . $urlArguments);
```
it may looks more clear